torch/_dynamo/utils.py

```diff
@@ -37,9 +37,7 @@
 from . import config, logging as torchdynamo_logging
 
 counters = collections.defaultdict(collections.Counter)
-troubleshooting_url = (
-    "https://github.com/pytorch/torchdynamo/blob/main/TROUBLESHOOTING.md"
-)
+troubleshooting_url = "https://github.com/pytorch/pytorch/blob/master/docs/source/dynamo/troubleshooting.rst"
```

Comment:
thanks for fixing this. the correct link is https://pytorch.org/docs/master/dynamo/troubleshooting.html